setup.py

```diff
@@ -13,4 +13,7 @@
         [console_scripts]
         colin=colin.cli.colin:cli
     ''',
+    data_files=[("/usr/share/colin/", ["colin/core/config/default.json",
```

Comment:
What about to move directory `colin/core/config` to `config`? Does it make sense?

Reply:
Yes, it would be useful for the users, that want to see them in the source.

Reply:
Moved.